x-pack/test/functional_endpoint/apps/endpoint/policy_list.ts

```diff
@@ -5,45 +5,86 @@
 
-  // FIXME: Skipped until we can figure out how to load data for Ingest
-  describe.skip('Endpoint Policy List', function() {
+  describe('When on the Endpoint Policy List', function() {
     this.tags(['ciGroup7']);
     before(async () => {
       await pageObjects.common.navigateToUrlWithBrowserHistory('endpoint', '/policy');
-      await pageObjects.endpoint.waitForTableToHaveData('policyTable');
     });
 
     it('loads the Policy List Page', async () => {
       await testSubjects.existOrFail('policyListPage');
     });
     it('displays page title', async () => {
-      const policyTitle = await testSubjects.getVisibleText('policyViewTitle');
+      const policyTitle = await testSubjects.getVisibleText('pageViewHeaderLeftTitle');
       expect(policyTitle).to.equal('Policies');
     });
     it('shows policy count total', async () => {
       const policyTotal = await testSubjects.getVisibleText('policyTotalCount');
-      expect(policyTotal).to.equal('100 Policies');
-    });
-    it('includes policy list table', async () => {
-      await testSubjects.existOrFail('policyTable');
+      expect(policyTotal).to.equal('0 Policies');
     });
     it('has correct table headers', async () => {
       const allHeaderCells = await pageObjects.endpoint.tableHeaderVisibleText('policyTable');
       expect(allHeaderCells).to.eql([
         'Policy Name',
-        'Total',
-        'Pending',
-        'Failed',
-        'Created By',
-        'Created',
-        'Last Updated By',
-        'Last Updated',
+        'Revision',
+        'Version',
+        'Description',
+        'Agent Configuration',
       ]);
     });
+    it('should show empty table results message', async () => {
+      const [, [noItemsFoundMessage]] = await pageObjects.endpoint.getEndpointAppTableData(
+        'policyTable'
+      );
+      expect(noItemsFoundMessage).to.equal('No items found');
+    });
+
+    describe('and policies exists', () => {
+      let policyInfo: PolicyTestResourceInfo;
+
+      before(async () => {
+        // load/create a policy and then navigate back to the policy view so that the list is refreshed
+        policyInfo = await policyTestResources.createPolicy();
+        await pageObjects.common.navigateToUrlWithBrowserHistory('endpoint', '/policy');
+        await pageObjects.endpoint.waitForTableToHaveData('policyTable');
+      });
+      after(async () => {
+        if (policyInfo) {
+          await policyInfo.cleanup();
+        }
+      });
+
+      it('should show policy on the list', async () => {
+        const [, policyRow] = await pageObjects.endpoint.getEndpointAppTableData('policyTable');
+        expect(policyRow).to.eql([
+          'Protect East Coast',
+          '1',
+          'Elastic Endpoint v1.0.0',
+          'Protect the worlds data - but in the East Coast',
+          policyInfo.agentConfig.id,
+        ]);
+      });
+      it('should show policy name as link', async () => {
```

Comment:
could we have tests like this be in unit tests?  seems like we could mock data in and test the view logic for link rendering, etc.  Not a big deal, just thinking about keeping functional tests smaller

Reply:
@kevinlog thats a good point. Yes, I think these should be covered via Unit test instead of Functional.

@aisantos Moving these to Unit will impact the code coverage as well so we will not loose those metrics.